File: algorithm/typescript/BOJ3046.ts
// https://www.acmicpc.net/problem/3046

import fs = require("fs");

const input: number[] = fs.readFileSync("/dev/stdin").toString().trim().split(" ").map(Number);

const R1: number = input[0];
const S: number = input[1];

console.log(2 * S - R1);

File: algorithm/java/src/BOJ3046.java
// problem : https://www.acmicpc.net/problem/3046

import java.util.Scanner;

public class BOJ3046 {
    static int R1, S;
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        R1 = sc.nextInt();
        S = sc.nextInt();

        System.out.println(2 * S - R1);
    }
}
